packages/hooks/src/__tests__/useTotalAssetCurrency.spec.tsx

```diff
@@ -0,0 +1,242 @@
+import * as React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useRealTotalAssetCurrency from '../useTotalAssetCurrency';
+
+describe('useRealTotalAssetCurrency', () => {
+    test('should return default currency when user has no account', async () => {
+        const mock = mockStore({
+            client: {
+                default_currency: 'EUR',
+            },
+        });
+
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mock}>{children}</StoreProvider>
+        );
+        const { result } = renderHook(() => useRealTotalAssetCurrency(), { wrapper });
+        expect(result.current).toBe('EUR');
```

Comment:
```suggestion
        expect(result.current).toBe(mock.client.default_currency);
```
@shayan-deriv You can reuse the `mock` variable as a constant when you need to check against it 🙇🏻